.github/CODEOWNERS

```diff
@@ -13,5 +13,12 @@
 # Cloud Posse must review any changes to GitHub actions
 .github/*     @cloudposse/engineering
 
-# Cloud Posse must review any changes to standard context definition
-**/context.tf   @cloudposse/engineering
+# Cloud Posse must review any changes to standard context definition,
+# but some changes can be rubber-stamped.
+**/context.tf   @cloudposse/engineering @cloudposse/approvers
+README.md       @cloudposse/engineering @cloudposse/approvers
+docs/*.md       @cloudposse/engineering @cloudposse/approvers
+
```

Comment:
Hrmmm this is changed in nearly every PR. We need to make sure `@contributors` can approve changes to the `README.md` and `docs/*`